src/app/course/goal-creation-page/goal-creation-page.component.html

```diff
@@ -0,0 +1,163 @@
+<h1 class="tui-text_h3 tui-space_top-10 tui-space_bottom-2">Create Goal</h1>
+<br>
```

Comment:
dont use <br>

All the styling should be either with taiga UI classes and if it cannot be achieved with taiga UI we use custom CSS.